etcdserver/server.go

```diff
@@ -632,6 +631,21 @@ func (s *EtcdServer) Lead() uint64 { return atomic.LoadUint64(&s.r.lead) }
 
 func (s *EtcdServer) Leader() types.ID { return types.ID(s.Lead()) }
 
+// maxRequestTimeout returns maximal operation timeout to finish.
+func (s *EtcdServer) maxOpTimeout() time.Duration {
```

Comment:
do you prefer it to be helper function as `func opTimeout(cfg Config) time.Duration`?